test/rules.js

```diff
@@ -283,6 +283,10 @@ var tests = {
         ,   { doc: "structure/canonical-missing.html", errors: ["structure.canonical.not-found"] }
         ,   { doc: "structure/canonical-href-missing.html", errors: ["structure.canonical.not-found"] }
         ]
+    ,   neutral: [
+            { doc: "structure/unneutral.html", warnings: ["structure.neutral.neutral", "structure.neutral.neutral", "structure.neutral.neutral", "structure.neutral.neutral"] }
```

Comment:
Since there's only one warning now, this test is failing.
```suggestion
            { doc: "structure/unneutral.html", warnings: ["structure.neutral.neutral"] }
```

We may also need more test files to test the different use cases.